tests/main/interfaces-desktop-document-portal/task.yaml

```diff
@@ -7,25 +7,22 @@ details: |
     portal needs to be mounted over $XDG_RUNTIME_DIR/doc inside the
     sandbox.
 
-systems: [-ubuntu-core-*]
+# Ubuntu 14.04 has special version of systemd which doesn't have StartTransientUnit API.
```

Comment:
this is fine since 14.04 is only supported for the livepatch snap, which doesn't use the desktop interface